### ISSUE TITLE
max() over a column of all-negative values returns zero

### ISSUE THREAD

**Issue by jonathanrhodes** [Gmousse/dataframe-js#46]
```
> dataFrame = new DataFrame.DataFrame([[-1], [-2], [-3]], ['value']);
> dataFrame.stat.max('value');
0
> dataFrame.stat.min('value');
-3
```

It appears that `max()` is initialized to 0, so it will never return a negative value: https://github.com/Gmousse/dataframe-js/blob/master/src/modules/stat.js#L36-L40.

**Comment by Gmousse** [comment]
Hey @jonathanrhodes .

Sorry for this newby bug, I fix it quickly.

**Comment by Gmousse** [comment]
@jonathanrhodes Closed and resolved in 1.2.8 (and in next 1.3.0)

**Comment by jonathanrhodes** [comment]
Thanks for the quick fix!

Jonathan Rhodes <https://www.linkedin.com/in/jonathanmrhodes> | VP of
Engineering
*metricstory.com <https://www.metricstory.com/> | @Metricstory
<https://twitter.com/MetricStory>*



On Sun, Apr 8, 2018 at 6:33 AM, Guillaume Mousnier <notifications@github.com
